### PR TITLE
Added build instructions to readme.

### PR DESCRIPTION
**Pull request by thefoojoo**
I wasn't able to find any instructions for building Jsonnet, so I wrote some. I'm new to the project, so I made some assumptions:
- Neither bazel nor make is favored over the other. I put make first since most users are likely to have it. This might be confusing for Windows users.
- Only g++ is required to build. Are there any other dependencies, e.g. operating system?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,47 @@
...
-Website: http://jsonnet.org
+For an introduction to Jsonnet and documentation,
+[visit our website](http://jsonnet.org).
 
-Discussion Forum: https://groups.google.com/forum/#!forum/jsonnet
+Visit our [discussion forum](https://groups.google.com/forum/#!forum/jsonnet).
+
+## Building Jsonnet
+
+The GCC C++ (g++) compiler is required to build Jsonnet.
```
Comment (sparkprime) [inline README.md]: clang is also supported

File: README.md [hunk 2]
```diff
@@ -2,6 +2,47 @@
...
+
+### Makefile
+
+To build jsonnet, run:
+
+```
+make -j
```
Comment (sparkprime) [inline README.md]: We should probably guard this with a -j 8 or something :)

File: README.md [hunk 3]
```diff
@@ -2,6 +2,47 @@
...
+```
+
+### Bazel
+
+Bazel builds are also supported.
+[Install Bazel](https://www.bazel.io/versions/master/docs/install.html) if it is
+not installed already. Then, run the following command to build:
+
+```
+bazel build //cmd:jsonnet
```
Comment (sparkprime) [inline README.md]: We should put an optimization flag in I think?  @davidzchen

File: README.md (hunk 2 above)
Comment (thefoojoo) [inline README.md]: I removed it. Doesn't take THAT long to build.

File: README.md (hunk 1 above)
Comment (thefoojoo) [inline README.md]: Done.

File: README.md (hunk 3 above)
Comment (thefoojoo) [inline README.md]: Done.

**Comment by sparkprime** [comment]
Sorry I forgot to come back to this.  Thanks for addressing my comments.  Did you mean to add the Eclipse stuff?

**Comment by sparkprime** [comment]
I forgot to chase you... :)  Shall I add the README without the other files (i.e. fork the PR)

**Comment by sparkprime** [comment]
Thanks, merged this in #293

**Pull request closed without merging**